### PR TITLE
feat(components): initia account detail

### PR DESCRIPTION
**Pull request by jennieramida**

### CODE REVIEW

File: public/font/pilatwide.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@font-face {
+    font-family: 'Pilat Wide';
+    src: url('../font/PilatWide-Bold.woff');
+    src: url('../font/PilatWide-Bold.woff2');
+    font-weight: 600;
+    font-style: normal;
+    }
```
Comment (songwongtp) [inline public/font/pilatwide.css:7]: please format the file

**Review by songwongtp — CHANGES_REQUESTED** [review]

File: src/lib/pages/account-details/components/modules/ModuleLists.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          <ModuleCard />
+          <ModuleCard />
+          <ModuleCard />
+          <ModuleCard />
+          <ModuleCard />
+          <ModuleCard />
+          <ModuleCard />
+        </SimpleGrid>
+      )}
+      {!isMobile && onViewMore && <ViewMore onClick={onViewMore} />}
```
Comment (poomthiti) [inline src/lib/pages/account-details/components/modules/ModuleLists.tsx:66]:
```
!isMobile && 
  (onViewMore ? ... : ...)
```

File: src/lib/pages/account-details/components/resources/ResourceLists.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      {!isMobile && (
+        // TODO: data
+        <SimpleGrid columns={{ sm: 1, md: 2, lg: 4 }} spacing={4} mb={6}>
+          <ResourceCard name="resource ja" amount={3} />
+          <ResourceCard name="resource ja" amount={3} />
+          <ResourceCard name="resource ja" amount={3} />
+          <ResourceCard name="resource ja" amount={3} />
+        </SimpleGrid>
+      )}
+      {!isMobile && onViewMore && <ViewMore onClick={onViewMore} />}
```
Comment (poomthiti) [inline src/lib/pages/account-details/components/resources/ResourceLists.tsx:65]:
```
!isMobile && (
<>
<SimpleGrid> .... </SimpleGrid>
{onViewMore && ...}
<>
)
```

File: src/lib/pages/account-details/components/resources/ResourceSection.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        <Flex direction="column" w="full">
+          <Flex alignItems="center" pb={6}>
+            <Heading as="h6" variant="h6">
+              0x1::dex
+            </Heading>
+            <Badge variant="primary" ml={2}>
+              9
+            </Badge>
+          </Flex>
+          <Flex>
```
Comment (poomthiti) [inline src/lib/pages/account-details/components/resources/ResourceSection.tsx:75]: Is this flex necessary?

File: src/lib/pages/account-details/index.tsx [hunk 5]
```diff
@@ -34,25 +35,30 @@ import { getFirstQueryParam, truncate } from "lib/utils";
...
 } from "./components/tables";
-import { TotalAccountValue } from "./components/TotalAccountValue";
 
 const tableHeaderId = "accountDetailsTab";
 
 enum TabIndex {
   Overview = "overview",
   Assets = "assets",
   Delegations = "delegations",
+  NFTs = "ntfs",
```
Comment (poomthiti) [inline src/lib/pages/account-details/index.tsx:55]: nfts

**Review by poomthiti — CHANGES_REQUESTED** [review]

**Review by songwongtp — CHANGES_REQUESTED** [review]

File: src/lib/pages/account-details/components/resources/ResourceSection.tsx (hunk 4 above)
Comment (jennieramida) [reply src/lib/pages/account-details/components/resources/ResourceSection.tsx:75]: yep 
added gap

File: src/lib/pages/account-details/components/modules/ModuleLists.tsx (hunk 2 above)
Comment (jennieramida) [reply src/lib/pages/account-details/components/modules/ModuleLists.tsx:66]: ไอพูมผิด

File: src/lib/pages/account-details/index.tsx [hunk 6]
```diff
@@ -187,6 +194,14 @@ const AccountDetailsBody = ({ accountAddress }: AccountDetailsBodyProps) => {
           <CustomTab onClick={handleTabChange(TabIndex.Delegations)}>
             Delegations
           </CustomTab>
+          <CustomTab
+            count={tableCounts.txsCount}
+            isDisabled={txCountLoading || tableCounts.txsCount === 0}
+            onClick={handleTabChange(TabIndex.NFTs)}
+            hidden={!move.enabled}
```
Comment (evilpeach) [inline src/lib/pages/account-details/index.tsx:201]: use nft config.

File: src/lib/pages/account-details/components/AccountHeader.tsx [hunk 7]
```diff
@@ -22,69 +24,95 @@ export const AccountHeader = ({
...
+            <Text fontWeight={500} color="text.dark" variant="body2">
+              HEX:
+            </Text>
+            {/* TODO: HEX */}
+            <CopyLink
+              value={accountAddress}
+              amptrackSection="account_top"
+              type="user_address"
+            />
+          </Flex>
```
Comment (evilpeach) [inline src/lib/pages/account-details/components/AccountHeader.tsx:81]: Let's enable this component when move.enabled is true

File: public/font/pilatwide.css [hunk 8]
```diff
@@ -0,0 +1,7 @@
+@font-face {
```
Comment (evilpeach) [inline public/font/pilatwide.css:1]: add `TODO: add another font face later` ?

File: src/config/chain/initia.ts [hunk 9]
```diff
@@ -32,6 +32,9 @@ export const INITIA_CHAIN_CONFIGS: ChainConfigs = {
       gov: {
         enabled: true,
       },
+      nft: {
+        enabled: false,
```
Comment (evilpeach) [inline src/config/chain/initia.ts:36]: true?

File: src/config/chain/neutron.ts [hunk 10]
```diff
@@ -63,8 +66,8 @@ export const NEUTRON_CHAIN_CONFIGS: ChainConfigs = {
       },
       wasm: {
         enabled: true,
-        storeCodeMaxFileSize: 800_000,
-        clearAdminGas: 50_000,
+        storeCodeMaxFileSize: 800000,
+        clearAdminGas: 50000,
```
Comment (evilpeach) [inline src/config/chain/neutron.ts:70]: Do not change

Comment (jennieramida) [reply src/config/chain/neutron.ts:70]: formatter change hai TT sad

**Review by evilpeach — APPROVED** [review]

**Pull request merged**